Makefile

```diff
@@ -0,0 +1,84 @@
+GO ?= go
+EPOCH_TEST_COMMIT ?= f2925f58acc259c4b894353f5fc404bdeb40028e
+PROJECT := github.com/kubernetes-incubator/cri-containerd
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
+GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
+PREFIX ?= ${DESTDIR}/usr/local
+BINDIR ?= ${PREFIX}/bin
+
+all: binaries
+
+default: help
+
+help:
+	@echo "Usage: make <target>"
+	@echo
+	@echo " * 'install' - Install binaries to system locations"
+	@echo " * 'binaries' - Build cri-containerd"
+	@echo " * 'clean' - Clean artifacts"
+	@echo " * 'lint' - Execute the source code linter"
```

Comment:
Can we replace this with a `verify` command to do all the verification all together?